README.md

```diff
@@ -51,6 +51,37 @@ run pod install
 pod install
 ```
 
+
+#### Swift Package Manager
+
+Add the dependency to your `Package.swift` and use in your target
+
+```swift
+dependencies: [
+    .package(url: "https://github.com/infinum/Locker.git", .upToNextMajor(from: "2.0.0"))
+]
+```
+
+Sample `Package.swift`
+
+```swift
+let package = Package(
+    name: "YourDependency",
+    products: [
+        .library(name: "YourDependency", targets: ["YourDependency"])
+    ],
+    dependencies: [
+        .package(url: "https://github.com/infinum/Locker.git", .upToNextMajor(from: "2.0.0")),
+    ],
+    targets: [
+        .target(
+            name: "YourDependency",
+            dependencies: [.product(name: "Locker", package: "Locker")]
+        )
+    ]
+)
+```
+
```

Comment:
```suggestion
To install Locker from the Swift Package Manager, you should:
* In Xcode 11+ select File → Packages → Add Package Dependency
* Enter project's URL: https://github.com/infinum/Locker.git

For more information, check [Swift Package Manager](https://swift.org/package-manager/) .
```

What you have described is addint the SPM as dependency into other spm libraries. Just change it with this as this is enough for addint the Locker into the project :)

Reply:
And consider removing the intro sentence: 
```
The easiest way to use Locker in your project is by using the CocoaPods package manager.
```

Also, check if you are missing this:
```
#### Carthage

For the Carthage installation and usage instruction, you can check official [quick start documentation](https://github.com/Carthage/Carthage#quick-start).

To integrate the library into your Xcode project, specify it in your `Cartfile`:

\```
github "infinum/Locker"
\```

Run `carthage update`.
```

Reply:
Added a new sentence under the Installation section to showcase that now we support CocoaPods, SPM and Carthage